test/Typewriter.Test/Metadata/MetadataMultipleNamespaces.xml

```diff
@@ -245,6 +245,7 @@
       <EntityType Name="singletonEntity1" BaseType="graph.entity">
         <NavigationProperty Name="testSingleNav" Type="graph.testType" ContainsTarget="true" />
       </EntityType>
+      <EnumType Name="EmptyEnum"/>
```

Comment:
Can you add the results of running this change so that we can see the results in https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/nikithauc/typecript-empty-enum/test/Typewriter.Test/TestDataTypeScript/com/microsoft/graph/src/Microsoft-graph.d.ts?

Reply:
Done